vibra/interface/project/save_project_data_selector.py

```diff
@@ -76,5 +78,15 @@ def proceed_callback(self):
     def closeEvent(self, a0: QCloseEvent | None) -> None:
         self.keep_window_open = False
         return super().closeEvent(a0)
+    
+
+def get_folder_size(path: Path):
+    total = 0
+    for dirpath, dirnames, filenames in os.walk(path):
+        for f in filenames:
+            fp = os.path.join(dirpath, f)
+            if os.path.isfile(fp):
+                total += os.path.getsize(fp)
+    return total
```

Comment:
```suggestion
def get_folder_size(path: Path):
    return sum(f.stat().st_size for f in path.rglob("*") if f.is_file())
```
Not sure if it sacrifices too much the legibility, it is a nice option